server/@types/prisonApi/index.d.ts

```diff
@@ -2344,7 +2339,7 @@ export interface components {
     /** HO Code */
     HOCodeDto: {
       /** Active Y/N */
-      activeFlag: 'N' | 'Y'
+      activeFlag: string
       /** HO code */
```

Comment:
This is generated from prison-API swagger isn't it?  Are these changes updates to prisonAPI itself?